src/spatio_temporal_voxel_grid.cpp

```diff
@@ -43,11 +43,12 @@ namespace volume_grid
 /*****************************************************************************/
 SpatioTemporalVoxelGrid::SpatioTemporalVoxelGrid(const float& voxel_size, \
                    const int& background_value, const int& decay_model,   \
-                   const double& voxel_decay) : 
+                   const double& voxel_decay, const bool& pub_voxels) :
                    _background_value( background_value ),                 \
```

Comment:
Please either remove the spaces prior / after background_value and voxel_size, or change them all to this. I probably did this.